#This message describes the robot endpoint state
float32 ee_z_pos    #the endpoint position
float32 ee_z_vel    #the endpoint velocity